docs/epidata_development.md

```diff
@@ -49,346 +49,122 @@ $ [sudo] make test pdb=1
-  ```
-
-  You can also run tests using pytest like this:
-  ```
-  docker run --network delphi-net --rm delphi_python pytest repos/delphi/delphi-epidata/integrations/
-  ```
-  and with pdb enabled like this:
-  ```
-  docker run --network delphi-net -it --rm delphi_python pytest repos/delphi/delphi-epidata/integrations/ --pdb
-  ```
-
-5. Bring down the servers, for example with the `docker stop` command.
-
-# rapid iteration
-
-The workflow described above requires containers to be stopped, rebuilt, and
-restarted each time code (including tests) is changed, which can be tedious. To
-reduce friction, it's possible to
-[bind-mount](https://docs.docker.com/storage/bind-mounts/) your local source
-files into a container, which replaces the corresponding files from the image.
-This allows your code changes to be reflected immediately, without needing to
-rebuild containers.
-
-There are some drawbacks, however, as discussed in the
-[Epicast development guide](https://github.com/cmu-delphi/www-epicast/blob/main/docs/epicast_development.md#develop).
-For example:
-
-- Code running in the container can read (and possibly also write) your local filesystem.
-- The command-line specification of bind-mounts is quite tedious.
-- Bind mounts do not interact well with `selinux` on some systems, leading to
-various access denials at runtime. As a workaround, you may have to use the
-[dangerous "Z" flag](https://docs.docker.com/storage/bind-mounts/#configure-the-selinux-label)
-or temporarily disable `selinux` -- neither of which is advised.
-
-## bind-mounting
-
-### non-server code
-
-Python sources (e.g. data acquisition, API clients, and tests), can be
-bind-mounted into a `delphi_python` container as follows:
+Note that the host and port given above are "external" values, which are
+locally visible. You'll need the `mysql` client installed locally to run the
+above command.
+
+In case you don't have the `mysql` client installed on your machine and
+don't want to install it, you can simply use the binary that's bundled with
+the `mariadb` docker image, which you should already have from building the
+`delphi_database` image. In that case, use the "internal" values, which are
+visible to containers on the same virtual network. For example:
 
 ```bash
-docker run --rm --network delphi-net \
-  --mount type=bind,source="$(pwd)"/repos/delphi/delphi-epidata,target=/usr/src/app/repos/delphi/delphi-epidata,readonly \
-  --mount type=bind,source="$(pwd)"/repos/delphi/delphi-epidata/src,target=/usr/src/app/delphi/epidata,readonly \
-  delphi_python \
-python3 -m undefx.py3tester.py3tester --color \
-  repos/delphi/delphi-epidata/integrations
+echo 'insert into fluview values \
+  (0, "2020-04-07", 202021, 202020, "nat", 1, 2, 3, 4, 3.14159, 1.41421, \
+  10, 11, 12, 13, 14, 15)' | \
+docker run --rm -i --network delphi-net mariadb \
+mysql --user=user --password=pass \
+  --port 3306 --host delphi_database_epidata epidata
```

Comment:
I appreciate the workaround, but I'm still getting an error

```
/usr/local/bin/docker-entrypoint.sh: line 713: exec: mysql: not found
```

Reply:
Ah yea, I managed to fix these. Edited the text around it to flow better too.